cursorless-talon/src/modifiers/scopes.py

```diff
@@ -46,6 +46,7 @@
     "selector": "selector",
     "state": "statement",
     "string": "string",
+    "branch": "branch",
```

Comment:
nit: any reason this is listed in this order in particular?

Reply:
Not really.  I should prob file a follow-up to alphabetise; didn't want to mess with the diff